lib/hex_web/owners.ex

```diff
@@ -0,0 +1,62 @@
+defmodule HexWeb.Owners do
```

Comment:
Personally I don't like `*Crud`, but it's hard to find an alternative. To me it kind of implies that all actions should be called c/r/u/d (similarly as in controllers, per convention) but we have better names that make sense in our domain: Key.generate, Release.publish etc. At the same time, obviously nothing stops us from using whatever function names we want in *Crud modules.

When I was experimenting (granted, only toy examples) with this kind of stuff (a layer between controllers, schemas & repo) I tried coming up with a module name that made sense in the domain, e.g. Shipping, Billing, Ledger, AddressBook etc, and when I failed I just pluralize it :) FWIW this particular module could just be called `Owner` (as it's not clashing with an existing schema name) and the remaining ones could be `HexWeb.{Docs, Packages, Releases, Users}`. I don't have a better suggestion than what I provided above, so I think I'll go with *Crud.

Btw, this module looks a lot like the layer between controllers & models that AFAIK is being discussed for "standardization" (for lack of a better word) in future versions of Phoenix. Perhaps there are already some ideas how to name this stuff and where to put it (e.g. *Crud & web/crud/*) @josevalim?